include/platform/mir/graphics/display_configuration.h

```diff
@@ -141,6 +141,10 @@ struct DisplayConfigurationOutput
 
     mir::optional_value<geometry::Size> custom_logical_size;
 
+    /** The output's displayable name. */
+    // TODO: Better output names that are consistant between sessions
+    std::string name = "OUT-" + std::to_string(id.as_value());
```

Comment:
This may not be the best approach to naming. The spec gives examples like:

> Examples of names include 'HDMI-A-1', 'WL-1', 'X11-1'

And, elsewhere, we use similar names:
```yaml
layouts:
# keys here are layout labels (used for atomically switching between them)
# when enabling displays, surfaces should be matched in reverse recency order
  default:                         # the default layout
    cards:
    # a list of cards (currently matched by card-id)
    - card-id: 0
      eDP-1:
        # This output supports the following modes: 2560x1440@60.0
        #
        # Uncomment the following to enforce the selected configuration.
        # Or amend as desired.
        #
        # state: enabled	# {enabled, disabled}, defaults to enabled
        # mode: 2560x1440@60.0	# Defaults to preferred mode
        # position: [0, 0]	# Defaults to [0, 0]
        # orientation: normal	# {normal, left, right, inverted}, defaults to normal
        # scale: 1
        # group: 0	# Outputs with the same non-zero value are treated as a single display
      DisplayPort-1:
        # (disconnected)
      HDMI-A-1:
        # (disconnected)
      DisplayPort-2:
        # (disconnected)
      HDMI-A-2:
        # This output supports the following modes: 1024x600@60.0, 1920x1080@60.0,
        # 1600x900@60.0, 1366x768@59.9, 1280x720@60.0, 1280x720@59.9
        #
        # Uncomment the following to enforce the selected configuration.
        # Or amend as desired.
        #
        # state: enabled	# {enabled, disabled}, defaults to enabled
        # mode: 1024x600@60.0	# Defaults to preferred mode
        # position: [0, 0]	# Defaults to [0, 0]
        # orientation: normal	# {normal, left, right, inverted}, defaults to normal
        # scale: 1
        # group: 0	# Outputs with the same non-zero value are treated as a single display
```

Reply:
Agreed. I've opened #2815 to address this.